src/main/scala/org/broadinstitute/dsde/firecloud/webservice/MethodsApiService.scala

```diff
@@ -38,7 +38,12 @@ trait MethodsApiService extends HttpService
         (get | post) {
           extract(_.request.method) { method =>
             extract(_.request.uri.query) { query =>
-              passthrough(Uri(passthroughBase).withQuery(query), method)
+              // only pass query params for GETs
```

Comment:
I am worried that we're doing some non-standard things with POSTs and query params, but I don't have any clear evidence on hand.

Reply:
we certainly don't document it in swagger for this endpoint ...